utils/joiner/joinpartition.cpp

```diff
@@ -481,8 +486,8 @@ int64_t JoinPartition::processSmallBuffer(RGData& rgData)
 
     ret = writeByteStream(0, bs);
 
-    if (rg.getRowCount())
-      htSizeEstimate += rg.getDataSize();
+    // FIXME(MCOL-5597): Properly calculate the size of the bucket.
```

Comment:
Are you sure this is the properly calculated size of the RG ? 
I mean the wording doesn't express the fact that Stringstore size is ignored. I suggest to elaborate on the fact that this is a heuristict and we ignore the Stringstore b/c the tests tells us to do so.

Reply:
Right, it's not a proper calculation and tag "FIXME" is added to specify this.
The reason of this patch  - @mariadb-LeonidFedorov asked to apply rollback based on @mariadb-SergeyZefirov research to pass 5597

Reply:
Ah, I misinterpreted `FIXME` part. 
Could you describe this FIXME in more words?